curriculum/challenges/english/25-front-end-development/lab-technical-documentation-page/587d78b0367417b2b2512b05.md

```diff
@@ -234,17 +252,13 @@ assert(cssCheck.length > 0 || htmlSourceAttr.length > 0);
 ```html
 <!DOCTYPE html>
 <html lang="en">
-
-<head>
-    <meta charset="UTF-8">
+  <head>
+    <meta charset="UTF-8" />
     <title>Technical Documentation Page</title>
-</head>
-
-<body>
-
-</body>
+  </head>
 
-</html> 
+  <body></body>
```

Comment:
maybe it's nicer to leave an empty line in the `body` in the seed code

Reply:
I didn't notice that line was removed. Putting it back now.